catalog/video/video-mpegts-stream/src/main/java/org/codice/alliance/video/stream/mpegts/netty/UdpStreamProcessor.java

```diff
@@ -208,11 +214,31 @@ public void visit(CreatedDateMetacardUpdater createdDateMetacardUpdater) {
             }
 
             @Override
-            public void visit(SecurityClassificationMetacardUpdater securityClassificationMetacardUpdater) {
+            public void visit(
+                    SecurityClassificationMetacardUpdater securityClassificationMetacardUpdater) {
 
             }
         });
 
+        streamEndPlugin.accept(plugin -> plugin.getFactory()
```

Comment:
Originally, I tried another approach for this, but it didn't work. In the blueprint file, I wanted to use the geometry operator beans for the stream end plugin and also use the same beans to create a top-level list of geometry operators to be stored in UdpStreamProcessor. This would eliminate the need to drill down through the stream end plugins when the geometry operators need to be updated. I've done this kind of thing with Spring, but I couldn't make it work for blueprint. If you want, we can sit down and try to design a better approach.